runtime/runtime/src/actions.rs

```diff
@@ -440,30 +441,39 @@ pub(crate) fn action_implicit_account_creation_transfer(
 ) {
```

Comment:
Can you add a comment to the method that it can only be used for near implicit accounts and/or rename it accordingly?